sw/device/lib/dif/dif_lc_ctrl_unittest.cc

```diff
@@ -166,17 +166,17 @@ TEST_F(AlertTest, NullArgs) {
 class MutexTest : public LcTest {};
 
 TEST_F(MutexTest, Acquire) {
-  EXPECT_WRITE32(LC_CTRL_CLAIM_TRANSITION_IF_REG_OFFSET, true);
-  EXPECT_READ32(LC_CTRL_CLAIM_TRANSITION_IF_REG_OFFSET, true);
+  EXPECT_WRITE32(LC_CTRL_CLAIM_TRANSITION_IF_REG_OFFSET, 0xa5);
+  EXPECT_READ32(LC_CTRL_CLAIM_TRANSITION_IF_REG_OFFSET, 0xa5);
   EXPECT_EQ(dif_lc_ctrl_mutex_try_acquire(&lc_), kDifLcCtrlMutexOk);
 
-  EXPECT_WRITE32(LC_CTRL_CLAIM_TRANSITION_IF_REG_OFFSET, true);
+  EXPECT_WRITE32(LC_CTRL_CLAIM_TRANSITION_IF_REG_OFFSET, 0xa5);
   EXPECT_READ32(LC_CTRL_CLAIM_TRANSITION_IF_REG_OFFSET, false);
```

Comment:
You can also update these to 0 to keep things consistent with the true encoding (0xa5). btw, is 0 -> 0x5a or some other value?

Reply:
I see, I will update to 0. Thanks for the suggestion. Yes from the documentation, it is 0 -> 0x5a. https://docs.opentitan.org/hw/ip/lc_ctrl/doc/index.html#Reg_claim_transition_if